### DISCUSSION TITLE
Set filters in collections in admin panel from URI

### DISCUSSION THREAD

**Discussion by 60pfennig** [payloadcms/payload#2905] [Roadmap]
Hi,
I'm trying to build an internal project managment tool which uses Payload as base and I tried to filter the collections by URI in the admin panel. Its working like its documented for the REST API but althought the results are filtered correctly the applied filter are not shown in the filter section of the UI. 
It would be really cool if they could be parsed from the URI!

**Comment by jmikrut** [comment]
great call, marking this as on the roadmap 👍
